src/runtime/HalideBuffer.h

```diff
@@ -437,7 +440,8 @@ class Buffer {
         buf.host = old_buf.host;
         buf.type = t;
         int d;
-        for (d = 0; d < 4 && old_buf.extent[d]; d++);
+        for (d = 0; d < 4 && old_buf.extent[d]; d++)
```

Comment:
A bit weird. Maybe the whole thing should be:

```
int d = 0;
while (d < 4 && old_buf.extent[d]) {
    d++;
}
```

Reply:
Done